src/platform/plugins/shared/unified_doc_viewer/public/components/observability/traces/components/similar_spans/index.tsx

```diff
@@ -13,12 +13,12 @@ import { DurationDistributionChart } from '@kbn/apm-ui-shared';
 import { ProcessorEvent } from '@kbn/apm-types-shared';
 import { ContentFrameworkChart } from '../../../../content_framework/chart';
 import { ContentFrameworkSection } from '../../../../content_framework/section';
-import type { SpanLatencyChartData } from '../../doc_viewer_span_overview/hooks/use_span_latency_chart';
+import type { LatencyChartData } from '../../doc_viewer_overview/hooks/use_latency_chart';
 
 export interface SimilarSpansProps {
   spanDuration: number;
   latencyChart: {
-    data: SpanLatencyChartData | null; // TODO move this interface
+    data: LatencyChartData | null; // TODO move this interface
```

Comment:
Is this comment still valid?

Reply:
I don't know on this one specifically, but I guess the interface is changed, so the todo is done?